### PR TITLE
Add the common content for the Python SDKs

### PR DESCRIPTION
**Pull request by shavidissa**
- Created a python folder
- Add the wavefrontsender.md and applicationtags.md files to the python folder
- Minor update to content.

### CODE REVIEW

**Comment by shavidissa** [comment]
cc @rkempf42

File: python/wavefrontsender.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+ 
+> **Note:** When you set up a Wavefront proxy on the specified proxy host, you specify the port it will listen to for each type of data to be sent. The `WavefrontProxyClient` must send data to the same ports that the Wavefront proxy listens to. Consequently, the port-related builder methods must specify the same port numbers as the corresponding proxy configuration properties. See the table given below:  
+
+| `WavefrontProxyClient()` parameter | Corresponding property in `wavefront.conf` |
+| ----- | -------- |
+| `metrics_port` | `pushListenerPorts=` |
+| `distribution_port` | `histogramDistListenerPorts=` |
+| `tracing_port` | `traceListenerPorts=` |
+
+## Option 2: Sending Data via Direct ingestion
```
Comment (rkempf42) [inline python/wavefrontsender.md]: Direct Ingestion (capital I)

File: python/wavefrontsender.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+When sending data via direct ingestion, you need to create a `WavefrontDirectIngestionClient`, and build it with the Wavefront URL and API token to send data directly to Wavefront.
+
+>**Prerequisite**
+> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-and-permissions).
```
Comment (rkempf42) [inline python/wavefrontsender.md:66]: This link is changing to Examine Groups, Roles, and Permissions (on my sandbox at http://docs-sandbox-a.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions . But, only when 56 goes out.

**Review by rkempf42 — APPROVED** [review]
Very minor comments. Otherwise, LGTM!

File: python/wavefrontsender.md (hunk 2 above)
Comment (shavidissa) [reply python/wavefrontsender.md:66]: Ack! Created a JIRA to track this.

**Comment by shavidissa** [comment]
CC @vikramraman

**Review by vikramraman — APPROVED** [review]

**Pull request merged**